metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '6.2.3'
+version           '6.2.4'
 source_url        'https://github.com/sous-chefs/haproxy'
```

Comment:
Please revert this change.

We'll bump this as part of a separate change.

Reply:
sure no problem